package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fhir-works-on-aws-deployment",
-  "version": "3.1.0",
+  "version": "4.0.0",
```

Comment:
Need update to user agent string https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/serverless.yaml#L57 as well

Reply:
Done